confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java

```diff
@@ -31,18 +32,39 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  * ConfigNodeStartupCheck checks the parameters in iotdb-confignode.properties and
  * confignode-system.properties when start and restart
  */
-public class ConfigNodeStartupCheck {
+public class ConfigNodeStartupCheck extends StartupChecks {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigNodeStartupCheck.class);
 
   private static final ConfigNodeConfig CONF = ConfigNodeDescriptor.getInstance().getConf();
 
+  public ConfigNodeStartupCheck(String nodeRole) {
+    super(nodeRole);
+  }
+
+  private void portCheck() throws StartupException {
+    Set<Integer> portSet = new HashSet<>();
+    int configNodePort = 2;
```

Comment:
Please do nothing use Hard Code here. A good solution is use static final params

Reply:
fixed